Assignment2/yncch99-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,39 @@
+{
+	"@context":{
+	"pr":"http://example.org/elements/properties",
+	"xsd":"http://www.w3.org/2001/XMLSchema#",	
+	"integer":"xsd:integer",
+	"dataTime":"xsd:dataTime",
+	"string":"xsd:string",
+	"includes":"pr:include",
+	"dimension":"pr:hasMeasurement",
+	"owner":"pr:hasOwner",
+	"temperature":"pr:hasTemperature",
+	"time":"pr:atTime",
+	"name":"pr:hasName",
+	"Class01":"pr:Class01",
+	"Sensor029":"pr:Sensor029",
+	"Measurement8401":"pr:Measurement8401",
+	"Computer101":"pr:Computer101",
+	"User10A":"pr:User10A"
```

Comment:
You are defining them as "@type" on your data, thus you don't need to define them here.